src/I18n/MessagesFileLoader.php

```diff
@@ -126,7 +125,7 @@ public function __invoke()
         }
 
         if (!$file) {
-            return $package;
+            return false;
```

Comment:
Will consumers of this class be ok with getting `false`? I think that's why the old code returned an empty `Package` instance.